os/pm/Kconfig

```diff
@@ -42,6 +43,60 @@ config PM_METRICS_DURATION
 
 endif
 
+comment "PM Options"
+
+config PM_NSTATE
+	int "Number of SoC sleep states"
+	default 0
+	---help---
+		Number of sleep states provided by the SoC Power module
+
```

Comment:
These below items can be displayed only if PM_NSTATE is greater than 1 like
if PM_NSTATE > 0
<all configs>
endif

Reply:
Done

Reply:
I agree Pradeep's opinion. ```> 0``` is better.

Reply:
Hello sunghan,
 Regarding this, I left review comment earlier 24 days ago. Please find it below,
 ```
lokeshbv 24 days ago Member

It doesn't work. please change it to PM_NSTATE != 0
As per Kconfig doc:

<expr> ::= <symbol>                             (1)
           <symbol> '=' <symbol>                (2)
           <symbol> '!=' <symbol>               (3)
           '(' <expr> ')'                       (4)
           '!' <expr>                           (5)
           <expr> '&&' <expr>                   (6)
           <expr> '||' <expr>                   (7)

Non-Constant symbols are not part of expressions (Non-constant symbols are defined with the 'config' statement.)
```
for `> 0`, following error occurs,
```
correct me if I'm wrong.
pm/Kconfig:55: syntax error
pm/Kconfig:19: missing end statement for this entry
Kconfig:483: missing end statement for this entry
pm/Kconfig:54: invalid statement
pm/Kconfig:101: unexpected end statement
pm/Kconfig:184: unexpected end statement
Kconfig:485: unexpected end statement
make: *** [Makefile.unix:579: menuconfig] Error 1
```

Reply:
You're right Thank you.